src/components/share-modal/share-modal.js

```diff
@@ -0,0 +1,28 @@
+import { connect } from 'react-redux';
+import get from 'lodash/get';
+
+import Component from './share-modal-component';
+
+const shareSocialMedia = [
+  {
+    link: `https://www.facebook.com/sharer/sharer.php?u=`,
+    className: 'facebookIcon',
```

Comment:
Here, instead of className as a string, you can import directly class from styles file:
```code
    import styles from './share-modal-styles.module';
```
```suggestion
    className: styles.facebookIcon
```